src/server/views/lists/partials/translators-interpreters/translators-interpreters-notice.njk

```diff
@@ -37,3 +37,7 @@
   If you have feedback specifically about this digital service, the FCDO welcomes your views. Email:
   <a class="govuk-link" href="mailto:digitalservicesfeedback@fco.gov.uk" class="govuk-link" target="_blank">digitalservicesfeedback@fco.gov.uk</a>
 </p>
+<h2 class="govuk-heading-m">Apply to be added to this service</h2>
+<p class="govuk-body">
+  You can <a class="govuk-link" href="{{ serviceApplyUrl }}">apply to be added to this service</a> if you are based outside the UK, and can provide translation or interpretation services to British nationals in English.
```

Comment:
spoke with Neil and agreed to remove the comma "outside the UK, and can provide".   Can you make this minor change.